packages/flex-plugin-scripts/src/scripts/start.ts

```diff
@@ -165,7 +193,23 @@ export const _updatePluginsUrl = (port: number) => {
 
   // Replace port and re-write to file
   plugins[pluginIndex].src = plugins[pluginIndex].src.replace(matches[1], port.toString());
-  fs.writeFileSync(paths().app.pluginsJsonPath, JSON.stringify(plugins, null, 2));
+  fs.writeFileSync(getPaths().app.pluginsJsonPath, JSON.stringify(plugins, null, 2));
+};
+
+/**
+ * Update port of a plugin
+ * @param port
+ * @param name
+ */
+export const _updatePluginPort = (port: number, name: string) => {
+  const config = readPluginsJson();
+  config.plugins.forEach((plugin) => {
+    if (plugin.name === name) {
+      plugin.port = port;
+    }
+  });
+
+  fs.writeFileSync(getPaths().cli.pluginsJsonPath, JSON.stringify(config, null, 2));
```

Comment:
We are using `fs.writeFileSync(..., JSON.stringify(...)` a lot. Create a wrapper for it in our `fs`:

```ts
const writeJSONFile = (path: string, obj: object) => fs.writeFileSync(path, JSON.stringify(obj, null, 2));
```

Then change all of these to be `fs.writeJSONFile(getPaths().cli.pluginsJsonPath, config);`

Reply:
I had to make the first parameter 'pth' instead because 'path' is a shadowed var